scipy/stats/tests/test_multivariate.py

```diff
@@ -513,6 +513,15 @@ def test_frozen(self):
         assert_allclose(norm_frozen.cdf(x), multivariate_normal.cdf(x, mean, cov))
         assert_allclose(norm_frozen.logcdf(x),
                         multivariate_normal.logcdf(x, mean, cov))
+    
+    def test_frozen_multivariate_normal_exposes_attributes(self):
```

Comment:
This can be [parametrized](https://docs.pytest.org/en/6.2.x/parametrize.html) over `cov`: in one case it should be a regular matrix; in the other, it should be a `Covariance` object.

Reply:
Done in c633842!